component/main.jsonnet

```
@@ -165,4 +167,31 @@ local clusterLoggingGroupVersion = 'logging.openshift.io/v1';
         policyTypes: [ 'Ingress' ],
       },
     },
+  '60_prometheus_rules':
+    kube._Object('monitoring.coreos.com/v1', 'PrometheusRule', 'syn-logging-rules')
+    {
+      metadata+: {
+        namespace: params.namespace,
+        labels: {
+          role: 'alert-rules',
+        },
+      },
+      spec: {
+        groups: [
+          {
+            name: 'syn-' + group.name,
+            rules: [
+              rule {
+                alert: 'SYN_' + rule.alert,
+                labels+: {
+                  syn: 'true',
```

We generally want to add a label indicating the component which manages the alert rules.

```suggestion
                  syn: 'true',
                  syn_component: 'openshift4-logging',
```